kod/object/passive/itematt/weapatt/waspellt.kod

```diff
@@ -191,8 +191,8 @@ messages:
       
       if send(self,@ReqAddToItem,#state1=iSpellType,#oItem=oWeapon)
       {
-         send(self,@AddToItem,#oItem=oWeapon, #state1=iSpellType,
-              #random_gen=TRUE);
+         send(self,@AddToItem,#oItem=oWeapon,#state1=iSpellType,
+              #timer_duration=86400000,#random_gen=TRUE);
```

Comment:
For big numbers like this it's generally better to do something like "60 * 60 * 24 * 1000" (or whatever) to make it clearer what's going on, or better yet, add a constant like "ONE_DAY_IN_MILLISECONDS" with that value.

Reply:
Makes sense. Did the same for another `AddToItem` call in this class, from which I borrowed this parameter.